src/index.html

```diff
@@ -9,6 +9,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+
+      <a href="#home">
+        <img 
+          class="icon" 
+          src="images/logo.png" 
+          alt="icon">
+      </a>
+
+      <nav class="nav">
+
+        <ul class="nav_list">
+          <li class="nav_link"><a class="link is-active" href="https://">Apple</a></li>
+          <li class="nav_link"><a class="link" href="https://">Samsung</a></li>
+          <li class="nav_link"><a class="link" href="https://">Smartphones</a></li>
+          <li class="nav_link">
+            <a 
+              class="link" 
+              data-qa="hover" 
+              href="https://"
+              >
```

Comment:
```suggestion
            <a 
              class="link" 
              data-qa="hover" 
              href="https://"
            >
```
Closing tag should be just under the opening